eglot-fsharp.el

```diff
@@ -64,10 +64,11 @@
 
 (defun eglot-fsharp--latest-version ()
   "Return latest fsautocomplete.exe version."
-  (let* ((json (with-temp-buffer (url-insert-file-contents "https://azuresearch-usnc.nuget.org/query?q=fsautocomplete&prerelease=false&packageType=DotnetTool")
-			         (json-parse-buffer)))
-         (versions (gethash "versions" (aref (gethash "data" json) 0))))
-    (gethash "version" (aref versions (1- (length versions))))))
+  (or eglot-fsharp--latest-version
+      (let* ((json (with-temp-buffer (url-insert-file-contents "https://azuresearch-usnc.nuget.org/query?q=fsautocomplete&prerelease=false&packageType=DotnetTool")
+			             (json-parse-buffer)))
+             (versions (gethash "versions" (aref (gethash "data" json) 0))))
+        (setq eglot-fsharp--latest-version (gethash "version" (aref versions (1- (length versions))))))))
```

Comment:
Use cache.

https://github.com/fsharp/emacs-fsharp-mode/blob/962f2157ed229d18e4f010ff5630a3ce4b4bdc67/eglot-fsharp.el#L63

The cache variable is declared but it is never used.

Reply:
The purpose of `(eglot-fsharp--latest-version)` is to ALWAYS return the exact version string, so we can compare the version string of the locally installed version string and prevent downloading the nuget artefact.

Maybe I should document this better.

Reply:
@juergenhoetzel Should I remove cache variable ? Comparison is already implemented. Sorry I didn't catch what you mean.

https://github.com/fsharp/emacs-fsharp-mode/blob/f8873096d24cac7b30419854347220f37345e780/eglot-fsharp.el#L77-L83